### PR TITLE
Implement Eve::AllianceList with version 1 argument

### PR DESCRIPTION
**Pull request by tslogan**
For #6.

This PR implements the case where version=1. The version=0 case will be implemented in a future PR.

### CODE REVIEW

**Comment by tslogan** [comment]
Ready for review

File: lib/greeve/eve/alliance_list.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+require_relative "../base_item"
+
+module Greeve
+  module Eve
+    # Returns active Alliances in New Eden and their member Corporations.
+    #
+    # @see https://eveonline-third-party-documentation.readthedocs.io/en/latest/xmlapi/eve/eve_alliancelist.html
+    class AllianceList < Greeve::BaseItem
+      endpoint "eve/AllianceList.xml.aspx"
```
Comment (amclain) [inline lib/greeve/eve/alliance_list.rb]: This part doesn't need the `.xml.aspx` part.

File: lib/greeve/eve/alliance_list.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+require_relative "../base_item"
+
+module Greeve
+  module Eve
+    # Returns active Alliances in New Eden and their member Corporations.
```
Comment (amclain) [inline lib/greeve/eve/alliance_list.rb]: `Alliances` and `Corporations` can be lowercase. I know this is straight from the docs, but their grammar is wrong.

**Review by amclain — CHANGES_REQUESTED** [review]

File: lib/greeve/eve/alliance_list.rb (hunk 2 above)
Comment (tslogan) [reply lib/greeve/eve/alliance_list.rb]: Updated

File: lib/greeve/eve/alliance_list.rb (hunk 1 above)
Comment (tslogan) [reply lib/greeve/eve/alliance_list.rb]: Updated

**Review by amclain — APPROVED** [review]

**Pull request merged**